test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+
+  test "No Benchmark Regression Detected" do
+    benchmark_run, initiator, benchmark_type, benchmark_result_type = setup_benchmarks
+    VCR.use_cassette('benchmark_regression') do
+      assert_nil BenchmarkRegressionJob.new.perform(benchmark_run, initiator, benchmark_type, benchmark_result_type)
+    end
+  end
+
+  test "Benchmark Regression Detected" do
+    benchmark_run, initiator, benchmark_type, benchmark_result_type = setup_benchmarks
+    benchmark_run.result = {"sometime"=>"5.1"}
+    VCR.use_cassette('benchmark_regression') do
+      assert_kind_of(Net::HTTPSuccess, BenchmarkRegressionJob.new.perform(benchmark_run, initiator, benchmark_type, benchmark_result_type))
+    end
+  end
+
+  private
+
+  def setup_benchmarks
+    benchmark_result_type = create(:benchmark_result_type)
```

Comment:
In Rspec, we prefer to use [let](https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let)